### DISCUSSION TITLE
blog/posts/2024-02-07-egm-results

### DISCUSSION THREAD

**Discussion by giscus** [redbrick/blog#8] [Blog]
# blog/posts/2024-02-07-egm-results

https://magma.redbrick.dcu.ie/blog/posts/2024-02-07-egm-results.html

**Comment by JedHazaymeh** [comment]
Welcome `@gholohan` :computer: :raised_hands:

**Comment by payne-2k3** [comment]
Awesome stuff Jed! This looks awesome :)

**Comment by browner1903** [comment]
Looks great!
